DESCRIPTION

```diff
@@ -16,7 +16,8 @@ Authors@R: c(
     person("David", "Freedman", role = "ctb"),
     person("F. Hoffmann-La Roche AG", role = c("cph", "fnd")),
     person("Cytel", role = c("cph", "fnd")),
-    person("Pfizer Inc.", role = c("cph", "fnd"))
+    person("Pfizer Inc.", role = c("cph", "fnd")),
+    person("Mahmoud", "Hamza", , "mahmoud.hamza@dataclin.com", role = "ctb")
```

Comment:
thanks @santarabantoosoo - one final minor comment: can you move this up above the 3 company names (as we usually have the funders come last) and you can take out your email address - we usually only add this for the maintainer. once done and man file updated then I'll approve and merge this one

Reply:
Done 

Thanks Ross